nucypher/crypto/powers.py

```diff
@@ -490,6 +497,38 @@ def encrypt_decryption_response(
             raise self.ThresholdResponseEncryptionFailed from e
 
 
+class SigningRequestDecryptingPower(ThresholdRequestPower):
```

Comment:
Let's leave "Decrypting" out of it so signing is separate from decryption. It's a little confusing with decryption in the name. wdyt?
```suggestion
class ThresholdSigningRequestPower(ThresholdRequestPower):
```

Reply:
✔️